### PR TITLE
Cross-mio updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Cruiseinformation - added Papa 9 and Irminger 9 cruises
Engineering Bulk load - added new stc board assembly SN due to swap of STC endcap on PMCO mooring (check config in Vault)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Papa 9 AM Updates [DT-161]](https://trello.com/c/n2lTwbTF/161-papa-9-am-updates-dt-161)

**Comment by rtravis-whoi** [comment]
[Irminger 9 AM Updates [DT-160]](https://trello.com/c/YNyU7qml/160-irminger-9-am-updates-dt-160)

**Comment by rtravis-whoi** [comment]
For Papa 9 cruise - see attached. 
Screenshot 2022-03-16 at 11-31-38 MFP Voyage Schedule

For Irminger 9 updates - taken from email from Sheri: 
AR69-01
Depart Woods Hole, MA 20 June
Arrive Reykjavik, Iceland 17 July

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -275,3 +275,5 @@ TS20220221,Tenacious II,2022-02-21T19:00:00,2022-02-22T16:00:00,Pioneer gliders
 EK20220225,Elakha,2022-02-25T13:45:00,2022-02-26T00:50:00,Endurance gliders 311 & 319 recoveries
 SKQ202205S,Sikuliaq,2022-03-21T23:20:00,2022-04-06T00:00:00,Endurance 16 moorings profilers and gliders
 AR66,Neil Armstrong,2022-04-08T00:00:00,2022-04-29T23:59:00,Pioneer 18
+SKQ202208S,Sikuliaq,2022-05-11T00:00:00,2022-05-30T23:59:00,Papa 9
+AR69-01,Neil Armstrong,2022-06-20T00:00:00,2022-07-17T23:59:00,Irminger 9
```
Comment (cdobs) [inline cruise/CruiseInformation.csv:279]: looks good

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -16,6 +16,7 @@ ASSET_UID,LEGACY_ASSET_UID,TYPE,Mobile,DESCRIPTION OF EQUIPMENT,MIO_Inventory_De
 3717-00004-00001-00007,,Sensor,0,Sensor and Telemetry Controller,Sensor and Telemetry Controller,WHOI,STC,3717-00004-00001-00007,,,,,Woods Hole Oceanographic Institution
 3717-00004-00001-00008,,Sensor,0,Sensor and Telemetry Controller,Sensor and Telemetry Controller,WHOI,STC,3717-00004-00001-00008,,,,,Woods Hole Oceanographic Institution
 3717-00004-00001-00010,,Sensor,0,Sensor and Telemetry Controller,Sensor and Telemetry Controller,WHOI,STC,3717-00004-00001-00010,,,,,Woods Hole Oceanographic Institution
+3717-00004-00001-00017,,Sensor,0,Sensor and Telemetry Controller,Sensor and Telemetry Controller,WHOI,STC,3717-00004-00001-00017,,,,,Woods Hole Oceanographic Institution
 3718-00001-00001-00008,,Sensor,0,Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3718-00001-00001-00008,,,,,Woods Hole Oceanographic Institution
 3718-00001-00001-00014,,Sensor,0,Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3718-00001-00001-00014,,,,,Woods Hole Oceanographic Institution
 3718-00001-00001-00046,,Sensor,0,Data Concentrator Logger,Data Concentrator Logger,WHOI,DCL,3718-00001-00001-00046,,,,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:22]: looks good

**Review by cdobs — APPROVED** [review]
Both cross-mio updates look good

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (sbeaulieu) [inline cruise/CruiseInformation.csv:279]: looks good

**Review by snwhite — APPROVED** [review]

**Pull request merged**